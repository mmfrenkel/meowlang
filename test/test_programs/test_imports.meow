GIMME IMPORT_EXAMPLE?

HAI ITZ ME NUMBR FUNC Main,

        PSST Test printing an integer
        PURR Say_Hello_World.

        GIVE 0.
KBYE
