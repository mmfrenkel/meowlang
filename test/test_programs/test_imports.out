

Say_Hello()
{
	char * username;
	char * message;
	username = Get_User_Name();
	message = "hello, " + username;
	printf("%X\n", message);
	Fstphrase();
	free(message);
	free(username);
}

int Main()
{
	Say_Hello();
	Sndphrase();
	Fstphrase();
	return 0;
}

Sndphrase()
{
	printf("%X\n", "My least favorite color is yellow");
	Get_User_Name();
}

char * Get_User_Name()
{
	char * username;
	printf("%X\n", "What is your name? ");
	Scan(username);
	return username;
}

Colors()
{
	char * green = "green";
	char * red = "red";
}

Fstphrase()
{
	printf("%X\n", "My favorite color is blue ");
}

int Main()
{
	Say_Hello();
	Sndphrase();
	Fstphrase();
	return 0;
}

